### ISSUE TITLE
Best model tracking operator

### ISSUE THREAD

**Issue by HeuristicLab-Trac-Bot** [heal-research/HeuristicLab#193]
*Issue migrated from trac ticket # 193*

**milestone:** HeuristicLab 3.3.0 | **component:** StructureIdentification | **priority:** medium | **resolution:** duplicate

#### 2008-07-04 15:35:52: @gkronber created the issue

___
> Like in HL2 the best models for the training set and the validation set should be tracked over the whole GP run.
